src/compiler/sys.ts

```diff
@@ -474,9 +478,9 @@ namespace ts {
         function getChakraSystem(): System {
 
             return {
-                newLine: "\r\n",
+                newLine: ChakraHost.newLine || "\r\n",

```

Comment:
@paulvanbrenk can you give this a quick try? would this work, or we need `"newLine" in CharkaHost` instead?